.github/workflows/content-lint-markdown.yml

```
@@ -19,44 +25,37 @@ jobs:
           # See https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable
           CHANGED_FILES: |-
-            ${{ steps.changed-files.outputs.all_changed_files }}
+            ${{ steps.changed_files.outputs.filtered_changed_files }}
         # If there are errors, using `--print-annotations` will make it
         # so it does *not* exit non-zero.
         # This is so that all warnings and errors are printed.
         run: npm run lint-content -- --print-annotations --paths $CHANGED_FILES
 
       - name: Run content linter if changed content/data files
-        if: steps.changed-files.outputs.any_changed == 'true'
+        if: steps.changed_files.outputs.any_changed == 'true'
```

actionlint (1.7.4)

property "any_changed" is not defined in object type {all_changed_files: string; filtered_changed_files: string}

(expression)